database.go

```diff
@@ -85,7 +85,8 @@ func createDatabase() {
 		last_name TEXT NOT NULL,
 		email TEXT NOT NULL UNIQUE,
 		username TEXT NOT NULL UNIQUE,
-		password TEXT NOT NULL
+		password TEXT NOT NULL,
+		user_type TEXT NOT NULL
```

Comment:
This is ok, but might cause many troubles. `user_type` cannot be any kind of text, it needs either to be `root` or `user`.
One way of achieving this would be to create yet another table `USERTYPE` with three columns `id` and `name` and `description`. An entry would be (1, 'user', 'Has access to the app'), for example.
you can leave it as it is for now.

Reply:
All right, however, I don't know how the new table could be used in order to link it with the USERS table. I prefer do this in the future.